src/Implementations/Redis/Legacy/Secure/EncryptedRedisStreamReader.cs

```diff
@@ -52,10 +52,13 @@ internal override async Task<List<EventMessage>> ProcessMessagesAsync(StreamEntr
                 try
                 {
                     string? content = msg.Content as string;
-                    byte[] encrypted = Convert.FromBase64String(content);
-                    byte[] plaintextBytes = await EncryptionHelpers.DecryptDataAsync(encrypted, _encryptionKey).ConfigureAwait(false);
-                    string plaintext = Encoding.UTF8.GetString(plaintextBytes);
-                    msg.Content = plaintext;
+                    if (content != null)
```

Comment:
can you look into the usage of the code and see if this should throw an exception or just set the msg.Content string

Reply:
We check if there are messages in RedisStreamReader, so content would only equal null if it is an empty message. I think we should set msg.Content to string.Empty if (content == null) rather than throw an error

Reply:
works for me